and-cli-workspace.js

```diff
@@ -64,19 +74,72 @@ require("./command-runner").run(async () => {
             }
 
             echo.success(`Cloned '${repo.full_name}'`);
+
+            configureNewClone(repo, prefix);
+
             results.successCount++;
         });
 
         return results;
     };
 
+    const configureNewClone = (repo, prefix) => {
+        echo.message(` - Configuring ${repo.full_name}...`);
+
+        // Additional configuration if its a clone of a fork
+        if (!repo.fork) {
+            return;
+        }
+
+        const folder = git.getCloneDirectoryName(repo.name, prefix);
+        shell.cd(folder);
+
+        const addUpgradeResult = git.addRemote(
```

Comment:
Question - is this meant to read `addUpstreamResult`?

Reply:
yup, who knows why i typed 'upgrade' 🤷